stimpacks/deploy/config.go

```diff
@@ -46,7 +48,9 @@ type Container struct {
 
 // Global describes global environment specs
 type Global struct {
-	Spec *Spec `yaml:"spec"`
+	Spec            *Spec  `yaml:"spec"`
+	RequiredVersion string `yaml:"requiredVersion"`
+	MinimumVersion  string `yaml:"minimumVersion"`
```

Comment:
I think this makes more sense in the `deployment` object section or maybe even as a top level item, what do you think?  The I think Global should maintain items that "cascade" down to the environments section which this shouldn't.

Reply:
I dont see how this could work in any other section, this does not really "cascade" like other things.  If part of the config requires a specific version the whole config needs it to be parsed correctly.

Reply:
There is a "deployment" section at the root level of the config that defines things about the deployment (like which docker container should be used, etc.). I think it makes more sense there. To my point, the this doesn't cascade whereas things in the global section are meant to.

Reply:
Ahh I miss read before.  It does not really matter to me, technically both "global" and "deploy" are top level objects and items in neither cascade down.  To me it still makes more sense in global, otherwise since Spec is the only thing that cascades.  I feel like it should be where you set "global" options that are not cascaded, though I see how one could treat deployment like that as well.  Lets have a quick chat offline but either way works for me in the end.